tests/settings.py

```diff
@@ -16,6 +16,7 @@
     'tests.test_us',
     'tests.test_pk',
     'tests.test_generic',
+    'tests.test_deprecated'
```

Comment:
Include a trailing comma so if more items are added to this list, we don't have to modify this line again.